clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt

```diff
@@ -341,6 +343,11 @@ open class MediaControl(core: Core, pluginName: String = name) : UICorePlugin(co
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+    protected fun setBackground(resource : Int) {
+        backgroundView.background = applicationContext.getDrawable(resource)
```

Comment:
We should use `ContextCompat` to avoid version compatibility issues.
```suggestion
        backgroundView.background = ContextCompat.getDrawable(applicationContext, resource)
```

Reply:
nice! I didn't know about `ContextCompact`. Thanks. :) done in 3af950f